Documentation/getting-started.md

```diff
@@ -45,9 +45,9 @@ The Edge Network extension depends on the following extensions:
 1. Add the Mobile Core, Identity for Edge Network, and Edge Network extensions to your project using the app's Gradle file:
 
 ```gradle
-implementation 'com.adobe.marketing.mobile:core:2.+'
-implementation 'com.adobe.marketing.mobile:edge:2.+'
-implementation 'com.adobe.marketing.mobile:edgeidentity:2.+'
+implementation 'com.adobe.marketing.mobile:core:3.+'
```

Comment:
let's use bom samples instead

Reply:
bom sample won't work until we release.  Should we keep it as 3.+ for now and update it along with build.gradle in sample apps all together before release?

Reply:
@cacheung We can update it now as it's our documentation and not the build config for an application. This doc won't go "live" until we release so it's okay to update it now.